components/CountdownTimer.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import Countdown from 'react-countdown'
```

Comment:
Add a test file for this component with basic tests

Reply:
test added